Tests/BankCardNumberTextInputFormatterTests.swift

```diff
@@ -0,0 +1,175 @@
+//
+//  BankCardNumberTextInputFormatterTests.swift
+//  TextInputKit
+//
+//  Created by Artem Starosvetskiy on 02/12/2016.
+//  Copyright © 2016 Artem Starosvetskiy. All rights reserved.
+//
+
+import XCTest
+@testable import TextInputKit
+
+private func textInputSimulator() -> TextInputSimulator {
+    return TextInputSimulator(BankCardNumberTextInputFormatter(.options()))
+}
+
+class BankCardNumberTextInputFormatterTests: XCTestCase {
+
+    func testThatUatpCardIsFormattedCorrectly() {
+        let textInput = textInputSimulator()
```

Comment:
Why do you not use 'textInput' as class variable using setUp and tearDown methods?